### ISSUE TITLE
GDPR compliance

### ISSUE THREAD

**Issue by XhmikosR** [jsdelivr/bootstrapcdn#1152]
TBF

**Comment by jdorfman** [comment]
@jmervine and I were talking and we should make sure we are protected.

- [ ] Possibly work with StackPath legal
- [ ] Make sure all of our partners are GDPR compliant
- [ ] Look into GDPR liability insurance
- [ ] Make clear differences between `www` and `cdn` in the privacy policy. E.g. files we serve are from a cookieless domain and all files (css, js, fonts) are unaltered.

**Comment by XhmikosR** [comment]
Personally, I was thinking these

1. Make sure out Policy page mentions everything (ZipRecruiter)
2. Add an informational cookie consent for starters
3. I can handle the opt-in behavior via JS for GA. But for the rest (GitHub button, ZipRecruiter), not sure how to do it
4. It seems Twitter doesn't save any cookies the way we use the timeline with `dnt` set to true.

**Comment by jdorfman** [comment]
You will have to educate @jmervine and I. While I can't speak for Josh, I can tell you I have no effing idea about this regulation. What I do know is I want us in compliance.

**Comment by XhmikosR** [comment]
Hey, I'm not an expert either :)

But as far as I understand it, it's not enough letting the visit know. We need to offer them a way to opt-out. Or rather opt-in, we need to confirm with legal.

So, for GA, it should be easy to opt in/out. For the other 2 cases I don't know.

**Comment by jdorfman** [comment]
via ZipRecruiter:

```
We have taken steps to ensure that we will be in compliance with GDPR regulations in a timely manner, and are doing so with the objective of keeping our ZipAlerts program in place with as few changes as possible consistent with our regulatory responsibilities.  
```

**Comment by jdorfman** [comment]
My contact at StackPath "no news yet, but I'm still on it"

**Comment by wttw** [comment]
So ... I'm temporarily in europe, and it seems none of my websites load correctly, as loading bootstrap from the CDN times out, traceroute to the ec2 node hosting it loops etc. It's loading fine from US locations. Is this intentional breakage in the name of not having to deal with GDPR, or just an outage?

**Comment by XhmikosR** [comment]
Totally irrelevant to GDPR. Check out the issue template and make a new
issue

**Comment by trish1400** [comment]
Hello,

I hope this is the appropriate place to post this.

We use BootstrapCDN (thanks!) and we've just installed Usercentrics as a Consent Management Platform.

It's identified that we are getting data from stackpath.bootstrapcdn.com and Usercentrics has categorised the service as follows:

```
**Service Name**: BootstrapCDN
**Description of Service**: This is a public content delivery network.
**Legal Basis**: Art. 6 para. 1 s. 1 lit. a GDPR
**Data Purposes**: Analytics, Advertising, Displaying content
**Data Collected**: Usage data, IP address, Browser information, Domain name, URL, Time of access or retrieval, Device operating system
**Data Recipients**: StackPath, LLCGoogle Ireland Limited
**Technologies Used**: Cookies
**Distribution to third countries**: United States of America
**Retention Period**: The data will be deleted as soon as they are no longer needed for the processing purposes.
**Name of Processing Company**: StackPath, LLC
**Address of Processing Company**: 2021 McKinney Avenue, Suite 1100, Dallas, TX 75201, United States of America
**Location of Processing**: United States of America
**Data Protection Officer**: privacy@stackpath.com
**Policy of Processor**: https://www.bootstrapcdn.com/privacy-policy/
```
My initial instinct was to categorise this BoostrapCDN as essential for the proper functioning of the site (so the user does not have the ability to opt-out) but this is problematic because  
- it says you're collecting IP address (which is personal data under GDPR)
- the Legal Basis is the consent clause (a)
- it states that as well as 'Displaying content' the data is used for 'Analytics' and 'Advertising'.

I was a bit surprised at this - so I thought I'd just ask if Usercentrics correct in this categorisation?  Are you using IP addresses for analytics and advertising?

Thanks!

**Comment by jimaek** [comment]
Hey, you can read more about how we use the data over here https://www.jsdelivr.com/terms/privacy-policy-jsdelivr-net

We also hired a law firm to review our service, you can read more about it over here as well
https://www.jsdelivr.com/blog/how-the-german-courts-ruling-on-google-fonts-affects-jsdelivr-and-why-it-is-safe-to-use/


TLDR: We don't use CDN IPs for advertising. We do use them for technical analytics, like download stats per project https://data.jsdelivr.com/v1/package/npm/jquery@3.2.1/stats but it's aggregated data and all IPs are purged within days.
